<!--xml-->
<launch>
    <node name="$(env ROS_HOSTNAME)_master_node" pkg="icarus_rover_v2" type="master_node"  output="screen"> 
        <param name="loop_rate"             value="500"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="false"/>    
    </node>
    <include file="$(find icarus_rover_v2)/launch/SensorLaunch/logitech_c270usbcam.launch" /> 
    <node name="$(env ROS_HOSTNAME)_cameramanager_node" pkg="icarus_rover_v2" type="cameramanager_node"  output="screen" clear_params="true">
        <param name="loop_rate"             value="500"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="false"/>    
        <param name="task_monitor1"         value="$(env ROS_HOSTNAME)_usb_cam"/>
    </node>
</launch>
